gamemodes/oresgyn/gamemode/cl_init.lua

```diff
@@ -7,6 +7,11 @@ include("rounds/cl_rounds.lua")
 include("economy/cl_economy.lua")
 include("cl_hud.lua")
 
+function GM:InitPostEntity()
+    net.Start(NET_PLAYER_JOIN)
+    net.SendToServer()
+end
+
```

Comment:
Sending the current round status to a new player didn't work properly.
Now it does.